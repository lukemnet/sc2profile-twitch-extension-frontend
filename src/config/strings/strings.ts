const strings = {
  config: {
    VALIDATION_URL_TOO_SHORT: 'Please provide correct StarCraft II profile URL.',
    VALIDATION_URL_INVALID: 'Profile URL is invalid.',
    VALIDATION_DUPLICATE_URLS: 'Duplicate profiles detected. Please review your configuration before saving.',
    SAVE_SUCCESS: 'Your configuration has been saved successfully. Please allow up to 30 minutes for changes to be visible in the extension.',
    SAVE_FAILURE: 'Failed to save your configuration due to server error. Please try again later.',
  },
};

export default strings;
