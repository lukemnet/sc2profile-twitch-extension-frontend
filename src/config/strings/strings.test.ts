import strings from './strings';

it('exports strings object', () => {
  expect(strings).toMatchSnapshot();
});
