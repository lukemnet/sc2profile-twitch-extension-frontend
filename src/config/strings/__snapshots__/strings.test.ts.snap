// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports strings object 1`] = `
Object {
  "config": Object {
    "SAVE_FAILURE": "Failed to save your configuration due to server error. Please try again later.",
    "SAVE_SUCCESS": "Your configuration has been saved successfully. Please allow up to 30 minutes for changes to be visible in the extension.",
    "VALIDATION_DUPLICATE_URLS": "Duplicate profiles detected. Please review your configuration before saving.",
    "VALIDATION_URL_INVALID": "Profile URL is invalid.",
    "VALIDATION_URL_TOO_SHORT": "Please provide correct StarCraft II profile URL.",
  },
}
`;
